_oasis

```diff
@@ -185,7 +185,7 @@ Library cli
                     CLIPager
   InternalModules:  CLIData, CLIPluginsLoaded
   CompiledObject:   byte
-  BuildDepends:     oasis, oasis.base, plugin-loader
+  BuildDepends:     oasis, oasis.base, plugin-loader, bytes
```

Comment:
On Thu, 30 Jul 2015 06:11:58 -0700, Sylvain Le Gall wrote:
> 
> In _oasis:
> 
> > @@ -185,7 +185,7 @@ Library cli
> >                      CLIPager
> >    InternalModules:  CLIData, CLIPluginsLoaded
> >    CompiledObject:   byte
> > -  BuildDepends:     oasis, oasis.base, plugin-loader
> > +  BuildDepends:     oasis, oasis.base, plugin-loader, bytes
> 
> This is actually problematic, because it means we have to depends on very recent version of OCaml (that provides the bytes library).

No, it only means that it will depend on a recent enough version of ocamlfind which provide an bytes module for older version of OCaml (it is an empty findlib package for more recent versions of the compiler).